src/main/kotlin/com/github/bartimaeusnek/modmixins/mixins/ztones/network/PackageFix.kt

```diff
@@ -0,0 +1,32 @@
+package com.github.bartimaeusnek.modmixins.mixins.ztones.network
+
+import com.riciJak.Ztones.network.ToggleMetaData
+import io.netty.buffer.ByteBufInputStream
+import net.minecraft.entity.player.EntityPlayer
+import org.spongepowered.asm.mixin.Mixin
+import org.spongepowered.asm.mixin.injection.At
+import org.spongepowered.asm.mixin.injection.Inject
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo
+import java.io.IOException
+
+@Mixin(value = [ToggleMetaData::class])
+class PackageFix {
+
+    @Throws(IOException::class)
+    @Inject(method = ["processData"], at = [At(value = "HEAD")], remap = false)
+    fun processData(c: CallbackInfo, entityPlayer: EntityPlayer, bbis: ByteBufInputStream) {
+        val itemStack = entityPlayer.heldItem
+        if (itemStack != null && itemStack.item?.javaClass?.name?.contains("com.riciJak.Ztones.block") == true) {
```

Comment:
Is this a copy of the original method? Would it work to check if itemstack is null or class isn't a ztone block and return? otherwise let the original method go?

Not a big deal though, this looks reasonable.

Reply:
Check can be more strict - ```com.riciJak.Ztones.item.block.ItemDecoBlocks2```